TrafficCapture/dockerSolution/build.gradle

```diff
@@ -116,4 +116,15 @@ task buildDockerImages {
     //dependsOn buildDockerImage_
 }
 
-tasks.getByName('composeUp').dependsOn(tasks.getByName('buildDockerImages'))
+tasks.getByName('composeUp').dependsOn(tasks.getByName('buildDockerImages').dependsOn('configureJavaVersion'))
+
+tasks.register('configureJavaVersion') {
```

Comment:
You're not doing any configuration here - you're checking the configuration.  You really should be doing the configuration though, so the name is good, the implementation should do more (see the overall note, this isn't the right place to do it)

Reply:
I see your point above about how if a user doesn't run the solution, they can still use a different version and can see how my solution is not valid since even though it only permits java 11 with the docker solution as a _whole_, separate tools can still be allowed to be built. I was somehow still focused on the docker solution and thought that this would be valid since it will technically cover each tool since it's hit before attempting to build them (in docker).
I think seeing #195 made me see the point clearly. Thanks for creating that! I'll go ahead and close this PR.